projects/components/src/overlay/sheet/sheet-overlay.component.scss

```diff
@@ -1,7 +1,13 @@
 @import 'color-palette';
 @import 'font';
 
-.htc-sheet {
+.sheet-overlay {
+  box-shadow: -3px 1px 24px rgba(0, 0, 0, 0.12), -1px -1px 8px rgba(0, 0, 0, 0.08);
```

Comment:
Same question here.

Reply:
Yes. It was like this from before. I will make them consistent in follow up pr